fuelup-init.sh

```diff
@@ -76,6 +78,36 @@ main() {
         esac
     done
 
+    if [ "$prompt_modify" = "yes" ]; then
```

Comment:
?
```suggestion
    if [ "$prompt_modify" -eq "yes" ]; then
```

Reply:
doesn't `-eq` test for numeric values only and `=` for strings? https://pubs.opengroup.org/onlinepubs/9699919799/utilities/test.html#tag_20_128_05